web3.js/src/connection.ts

```diff
@@ -711,11 +711,17 @@ export type SimulatedTransactionAccountInfo = {
   rentEpoch?: number;
 };
 
+export type SimulatedTransactionReturnData = {
+  programId: string;
+  data: string[];
```

Comment:
The return data is actually given as a tuple of `(String, encoding)`: https://github.com/solana-labs/solana/blob/128226c6ccbd1071702e5140f01b3ce21011d722/client/src/rpc_response.rs#L409 -- so could this be
```suggestion
  data: [string, TransactionReturnDataEncoding];
```

And also have:
```
export type TransactionReturnDataEncoding = 'base64';
```